urllib3/poolmanager.py

```diff
@@ -142,7 +137,7 @@ def connection_from_url(self, url):
         u = parse_url(url)
         return self.connection_from_host(u.host, port=u.port, scheme=u.scheme)
 
-    def urlopen(self, method, url, redirect=True, **kw):
+    def urlopen(self, method, url, redirect=True, retries=None, **kw):
```

Comment:
I decided to leave `retries` in the signature for `PoolManager.urlopen`; if we let it filter into `**kw`, we'd have to pop it anyway, as .redirect requires a true `Retries` object to be passed to it, and the one in `**kw` might be an int. In the previous version, this didn't matter, as the redirect logic wasn't a separate function we were passing `**kw` to.